install-devstack-xen.sh

```diff
@@ -4,11 +4,12 @@ set -eu
 function print_usage_and_die
 {
 cat >&2 << EOF
-usage: $0 XENSERVER XENSERVER_PASS PRIVKEY <optional arguments>
+usage: $0 XENSERVER XENSERVER_PASS PRIVKEY NODE_TYPE NODE_NAME [CONTROLLER_IP] <optional arguments>
```

Comment:
CONTROLLER_IP is optional ATM and I can make all the three optional, but I don't want to make the using -x -y -z flag